i18n/nitrokey_pl.ts

```diff
@@ -468,7 +468,7 @@ Spróbuj ponownie</translation>
     </message>
     <message>
         <source>Base32</source>
-        <translation>Base32</translation>
+        <translation>Podstawowy 32-bitowy</translation>
```

Comment:
`Base32` is an own name of a coding - it is OK to not translate and leave it as is, since translation will introduce confusion.